infrastructure/calibratenoise/src/utils.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+class TSVHandler:
```

Comment:
calibratenoise.py already writes the TSV file, no need to reimplement that in a different way.